src/network/chart_cache_updater.py

```diff
@@ -271,9 +290,17 @@ def _overwrite_song_name(expanded_song_list):
     db.cachedb.commit()
 
 
+timers = [(7, 4.5, 'h'), (9, 6, 'h'), (11, 7.5, 'h'), (12, 7.5, 'm'),
+          (6, 4.5, 'm'), (9, 7.5, 'm'), (11, 9, 'm'), (13, 9, 'h')]
```

Comment:
Variable name for final static-ish type should be in ALL_CAPS. Also should move to top of the file.